src/Components/ReflectionForm/ReflectionForm.js

```diff
@@ -1,17 +1,54 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types'
+import { postAReflection } from '../../apiCalls.js'
 import './ReflectionForm.scss';
 
-const ReflectionForm = () => {
+class ReflectionForm extends Component {
+ constructor(props){
+   super(props)
+   this.state={
+     reflection: '',
+   }
+ }
 
-  return (
-    <section>
-      <h1>Reflection Form</h1>
+ keepReflection = (e) => {
+   this.setState({reflection: e.target.value})
+ }
+
+ submitReflection = (e) => {
+   e.preventDefault()
+   postAReflection(this.state.reflection, new Date(), this.props.mood)
+   document.getElementById("textArea").value = ''
+ }
+  render() {
+    return (
+      <section className='ReflectionForm'>
+      <form data-testid='reflectionForm'>
+      <h2  className='reflectionPrompt' data-testid='reflectionPrompt'> You're feeling fine, what went well today? </h2>
+      <label>
+      Write your reflection here:
+      <textarea
+      id='textArea'
+      data-testid='reflectionArea'
```

Comment:
This testid comes in SO handy with testing. Glad we're implementing it!

Reply:
Agreed! Yay job shadows and getting tips and tricks from pros. I really feel like we've leveled up regarding streamlining our testing process.